packages/devtools_app/lib/src/ui/tab.dart

```diff
@@ -19,13 +19,13 @@ class DevToolsTab extends Tab {
   /// The only difference is this tab makes more of an effort to reflect
   /// changes in font and icon sizes.
   DevToolsTab._({
-    Key key,
-    String text,
-    Icon icon,
+    required Key? key,
```

Comment:
required and non-nullable. The only place this should be constructed is from the factory constructor, and we know it is non-null.

Reply:
done